types/react-swipeable-views-utils/index.d.ts

```diff
@@ -0,0 +1,25 @@
+// Type definitions for react-swipeable-views-utils 0.13
+// Project: https://github.com/oliviertassinari/react-swipeable-views#react-swipeable-views
+// Definitions by: Sebastian Silbermann <https://github.com/eps1lon>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 3.0
+
+import * as React from 'react';
+import { ConsistentWith, Omit, PropInjector } from '@material-ui/types';
```

Comment:
Doesn't look they are used:

```suggestion
import { PropInjector } from '@material-ui/types';
```